src/component/organism/listitem/AnimalInfo.js

```diff
@@ -3,8 +3,7 @@ import {Text, View} from 'react-native';
 import {GRAY20} from 'Root/config/color';
 import {txt} from 'Root/config/textstyle';
 import PetImageLabel from 'Molecules/label/PetImageLabel';
-import {animalInfo} from 'Organism/style_organism';
-
+import {animalInfo} from '../style_organism copy';
```

Comment:
import 시에 상대참조보다 절대참조로 가도록 변경 'Organism/style_organism copy'로 변경해야함
```suggestion
import {animalInfo} from 'Organism/style_organism copy';
```